src/test/java/uk/gov/hmcts/probate/service/exceptionrecord/mapper/OCRFieldAdditionalExecutorsApplyingMapperTest.java

```diff
@@ -72,6 +87,68 @@ public void setUpClass() throws Exception {
+                .executorsApplying2applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .build();
+
+        ocrFields5 = ExceptionRecordOCRFields.builder()
+                .executorsApplying0applyingExecutorName(EXECUTOR_APPLYING_1_NAME)
+                .executorsApplying0applyingExecutorOtherNames(EXECUTOR_APPLYING_1_OTHER_NAMES)
+                .executorsApplying0applyingExecutorEmail(EXECUTOR_APPLYING_1_EMAIL)
+                .executorsApplying0applyingExecutorAddressLine1(EXECUTOR_APPLYING_1_ADDRESS_LINE1)
+                .executorsApplying0applyingExecutorAddressLine2(ADDRESS_LINE2)
+                .executorsApplying0applyingExecutorAddressTown(ADDRESS_POST_TOWN)
+                .executorsApplying0applyingExecutorAddressCounty(ADDRESS_COUNTY)
+                .executorsApplying0applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .executorsApplying1applyingExecutorName(EXECUTOR_APPLYING_2_NAME)
+                .executorsApplying1applyingExecutorOtherNames(EXECUTOR_APPLYING_2_OTHER_NAMES)
+                .executorsApplying1applyingExecutorEmail(EXECUTOR_APPLYING_2_EMAIL)
+                .executorsApplying1applyingExecutorAddressLine1(EXECUTOR_APPLYING_2_ADDRESS_LINE1)
+                .executorsApplying1applyingExecutorAddressLine2(ADDRESS_LINE2)
+                .executorsApplying1applyingExecutorAddressTown(ADDRESS_POST_TOWN)
+                .executorsApplying1applyingExecutorAddressCounty(ADDRESS_COUNTY)
+                .executorsApplying1applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .executorsApplying2applyingExecutorName(EXECUTOR_APPLYING_3_NAME)
+                .executorsApplying2applyingExecutorOtherNames(EXECUTOR_APPLYING_3_OTHER_NAMES)
+                .executorsApplying2applyingExecutorEmail(EXECUTOR_APPLYING_3_EMAIL)
+                .executorsApplying2applyingExecutorAddressLine1(EXECUTOR_APPLYING_3_ADDRESS_LINE1)
+                .executorsApplying2applyingExecutorAddressLine2(ADDRESS_LINE2)
+                .executorsApplying2applyingExecutorAddressTown(ADDRESS_POST_TOWN)
+                .executorsApplying2applyingExecutorAddressCounty(ADDRESS_COUNTY)
+                .executorsApplying2applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .executorsApplying3applyingExecutorName(EXECUTOR_APPLYING_4_NAME)
+                .executorsApplying3applyingExecutorOtherNames(EXECUTOR_APPLYING_4_OTHER_NAMES)
+                .executorsApplying3applyingExecutorEmail(EXECUTOR_APPLYING_4_EMAIL)
+                .executorsApplying3applyingExecutorAddressLine1(EXECUTOR_APPLYING_4_ADDRESS_LINE1)
+                .executorsApplying3applyingExecutorAddressLine2(ADDRESS_LINE2)
+                .executorsApplying3applyingExecutorAddressTown(ADDRESS_POST_TOWN)
+                .executorsApplying3applyingExecutorAddressCounty(ADDRESS_COUNTY)
+                .executorsApplying3applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .build();
```

Comment:
what are these objects building?

Reply:
these are the builds for testing the 3rd and 4th applying executors